docs/content/en/integrations/importing.md

```diff
@@ -118,4 +118,23 @@ A classic way of reimporting a scan is by specifying the ID of the test instead:
     "scan_type": 'ZAP Scan',
     "test": 123,
 }
-```
\ No newline at end of file
+```
+
+## Using the `scan_date` field
+
+DefectDojo offers a plethora of supported scanner reports, but not all of them contain the 
+information most important to a user. The `scan_date` field is a flexible smart feature that
+allows users to set the completion date of the a given scan report, and have it propagate
+down to all the findings imported. This field is **not** mandatory, but the default value for 
+this field is the date of import (whenever the request is processed and a successful response is returned). 
+
+Here are the following use cases for using this field:
+
+1. The report **does not** set the date, and `scan_date` is **not** set at import
+    - Finding date will be the default value of `scan_date`
```

Comment:
What is the "default value of scan_date" when it's not set? If it's the current date, then it should say "Finding date will be current date"

Reply:
This is in the lat sentence of the first paragraph